vcpkg/commands/test-features.md

```diff
@@ -0,0 +1,68 @@
+---
+title: vcpkg test-features
+description: Command line reference for the vcpkg test-features command. Builds all features of a package.
+ms.date: 08/15/2023
+---
+
+# vcpkg test-features
+
+## Synopsis
+
+```console
+vcpkg test-features [options] <package>...
+```
+
+## Description
+
+Tests all features of a package by installing them in manifest mode. This command is useful for testing that all features of a package are working correctly.
+The following tests are performed:
+- The core feature gets installed with default featured disabled
+- Every feature gets installed with default featured disabled
+- All features combined gets installed
```

Comment:
How this command deal with unsupported features, or features with unsupported dependencies?

Reply:
Unsupported features are not tested, features with unsupported dependencies must be marked as cascade. 

I have thought of testing unsupported features to report if they build successfully (with a baseline entry to prevent this test. For example for vcpkg-msbuild)

Reply:
I would like to see some clear wording about this, e.g. "Feature which are unsupported or marked as cascade for the target triplet are skipped."